babel.config.js

```diff
@@ -31,24 +31,32 @@ const argv = minimist(process.argv.slice(2));
 
 // eslint-disable-next-line amphtml-internal/no-module-exports
 module.exports = function(api) {
+  const isDist = argv._.includes('dist');
+  const {esm} = argv;
+  const es5Target = {
+    'browsers': isTravisBuild()
+      ? ['Last 2 versions', 'safari >= 9']
+      : ['Last 2 versions'],
+  };
+
+  const esmTarget = {
+    'esmodules': true,
+  };
+
   api.cache(true);
-  // `dist` builds do not use any of the default settings below. (Both
-  // Multipass and Singlepass)
-  if (argv._.includes('dist')) {
+  // `dist` builds do not use any of the default settings below until its
+  // an esm build. (Both Multipass and Singlepass)
+  if (isDist && !esm) {
     return {};
   }
   return {
     'presets': [
       [
-        '@babel/env',
+        '@babel/preset-env',
```

Comment:
@erwinmombay @jridgewell – Looks like this wasn't using `preset-env` before, would this of worked previously?